IronFoundry.Container.Test/ContainerTests.cs

```diff
@@ -227,6 +227,20 @@ public void WhenProcessEnvironmentConflictsWithContainerEnvironment()
 
                     Assert.Equal(Spec.Environment, actualSpec.Environment);
                 }
+
+                [Fact]
+                public void WhenPathMappingDisabled_DoesntMapWorkingDir()
```

Comment:
For `@ROOT@` handling on the Warden side, can we get rid of the `@ROOT@` prefix and use the ContainerDirectory to map the user path?  Instead of doing any mapping in the Warden.